std_msgs/Float32MultiArray screws # centers
std_msgs/Float32MultiArray screws_near_cpu # centers
std_msgs/Float32MultiArray back_cover_cut_path # cut path
std_msgs/Float32MultiArray motherboard # gripping points
std_msgs/Float32MultiArray fan # center
std_msgs/Float32MultiArray cpu # no use for now
std_msgs/Float32MultiArray keyboard # center
std_msgs/Float32MultiArray front_cover # gripping points
std_msgs/Float32MultiArray keyboard_cut_path # cut path
std_msgs/Float32MultiArray front_cover_cut_path # cut path
std_msgs/Float32MultiArray cd_rom # center
std_msgs/Float32MultiArray hard_disk # center
std_msgs/Float32MultiArray flipping_points # points
std_msgs/Float32MultiArray[] ports_cut_path # cut path(s)
std_msgs/Float32MultiArray[] screws_cut_path # cut path(s)
std_msgs/Float32MultiArray[] screws_near_cpu_cut_path # cut path(s)
std_msgs/Float32MultiArray[] cd_rom_cut_path # cutting line(s)